app/components/classEdit/classEdit.controller.js

```diff
@@ -36,18 +31,25 @@ angular
 		vm.editClass = editClass;
 		
 		vm.cancel = cancel;
+        
+        activate();
+        
+        function activate(){
+            if(singleClass.name == ''){
+                vm.addingOption = true;
```

Comment:
I don't think it's necessary at all, but if you want to have it then rename to vm.editMode.